tool/sel4coreplat/__main__.py

```diff
@@ -1486,6 +1503,7 @@ def build_system(
     for pd in system.protection_domains:
         # Could use pd.elf_file.write_symbol here to update variables if required.
         pd_elf_files[pd].write_symbol("sel4cp_name", pack("<16s", pd.name.encode("utf8")))
+        pd_elf_files[pd].write_symbol("passive", pack("?", pd.passive))
```

Comment:
What does `?` do in this context?

Reply:
`?` is for a boolean type with a size of a single byte (as per https://docs.python.org/3/library/struct.html#format-characters )